src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -341,7 +335,7 @@ private function doPurge(string $url)
     /**
      * Loads data for the given key.
      *
-     * @return string The data associated with the key
+     * @return string|false The data associated with the key
```

Comment:
IMO we should change this to a nullable string return value

Reply:
Fixed in #33007.